src/main/scala/mesosphere/marathon/api/akkahttp/Rejections.scala

```diff
@@ -17,14 +17,19 @@ object Rejections {
   case class EntityNotFound(message: Message = defaultEntityNotFoundMessage) extends Rejection
   object EntityNotFound {
     def noApp(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
-      EntityNotFound(Message(s"App '$id' does not exist" + version.fold("")(v => s" in version $v")))
+      val readableVersion = version.fold("")(v => s" in version $v")
+      EntityNotFound(Message(s"App '$id' does not exist $readableVersion"))
     }
     def noGroup(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
-      EntityNotFound(Message(s"Group '$id' does not exist" + version.fold("")(v => s" in version $v")))
+      val readableVersion = version.fold("")(v => s" in version $v")
```

Comment:
What do you think if we factor this to a common method?

Reply:
Yeah, let's do it.